6/melinda-seckington-captions.srt

```diff
@@ -1204,15 +1206,15 @@ Someone who grabbed a question.
 
 283
 00:23:11,384 --> 00:23:14,480
-Let's just inaudible for now, plus the work.
+Let's just do that for now, does that work?
 
 284
 00:23:15,616 --> 00:23:19,352
-Yes, it would if I did that inaudible
+Yes, it would if I did that [inaudible]
```

Comment:
I think we can drop this `[inaudible]` and end this line with an exclamation mark.